#!/usr/bin/python3
def raise_exception_msg(message=""):
    raise NameError(message)


if __name__ == '_main__':
    raise_exception_msg(message="")
